event-adapter/CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing guidelines
```

Comment:
Please remove files: contributing, licence, code-of-conduct. Equivalent files are in this repo's root directory.